### PR TITLE
style blockquotes business ethics

### PR DESCRIPTION
**Pull request by ajluk**
Issue:  #2822

### CODE REVIEW

File: styles/books/business-ethics/book.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@ $PagesWithBands: (
   (pageName: appendix, firstSelector: '.appendix', generalSelector: '.appendix'),
 );
 
+
```
Comment (Stackblocks) [inline styles/books/business-ethics/book.scss:7]: Any reason for the added line here?

File: styles/templates/template3/_design.scss [hunk 2]
```diff
@@ -10,3 +10,6 @@
 @import 'designs/common/pdf/page';
 @import 'designs/common/reset';
 @import 'designs/common/pdf/footnotes-shapes.scss';
+
```
Comment (Stackblocks) [inline styles/templates/template3/_design.scss:13]: If this is a gap for the purposes of separation could you include a comment about what differentiates the groups?

File: styles/templates/template3/_settings.scss [hunk 3]
```diff
@@ -1,6 +1,13 @@
+//global template settings
 @import "colors";
 @import "typography";
 
+
+//template parts
+@import 'templates/template3/parts/link-settings';
+
```
Comment (Stackblocks) [inline styles/templates/template3/_settings.scss:8]: Extra spacing here as well

**Review by Stackblocks — CHANGES_REQUESTED** [review]
Looks good aside from some spare newlines here and there :+1:

File: styles/output/business-ethics-pdf.css [hunk 4]
```diff
@@ -688,4 +688,11 @@ sub {
   prince-image-resolution: auto, 200dpi;
   prince-background-image-resolution: 200dpi; }
 
+:root {
+  margin-left: 0.25inch;
+  margin-right: 0.25inch; }
```
Comment (wilkus27) [inline styles/output/business-ethics-pdf.css:693]: Somethings went wrong :thinking: Settings are added to the root not to blockquote.

**Review by wilkus27 — CHANGES_REQUESTED** [review]

File: styles/books/business-ethics/book.scss (hunk 1 above)
Comment (ajluk) [reply styles/books/business-ethics/book.scss:7]: I think I have pasted the settings first here by accident and left the line empty after removing it. Thanks!

File: styles/templates/template3/_design.scss (hunk 2 above)
Comment (ajluk) [reply styles/templates/template3/_design.scss:13]: Yes, I just wanted to separate the shapes that are inside the `parts` folder from the others,  common/pdf shapes as we did for the previous themes.

File: styles/output/business-ethics-pdf.css (hunk 4 above)
Comment (ajluk) [reply styles/output/business-ethics-pdf.css:693]: @wilkus Thank you for catching it up!

**Review by Stackblocks — APPROVED** [review]

**Review by wilkus27 — APPROVED** [review]

**Pull request merged**